### PR TITLE
Identation 2

### PR DESCRIPTION
**Pull request by p-steuer**
Rebased PR https://github.com/opencryptoki/opencryptoki/pull/129 and integrated @ifranzki  s feedback so @dodys dont has to do more work ; -)

### CODE REVIEW

File: testcases/crypto/des3.h [hunk 1]
```diff
@@ -862,50 +856,50 @@ static struct des3_test_vector des3_ofb64_tv[] = {
 #endif
```
Comment (ifranzki) [inline testcases/crypto/des3.h]: According to the new coding style the "# define NUM_OF_PUBLISHED_TESTSUITES" should probably be without a blank.
I should have made it the correct was with my patch already, sorry....

**Comment by p-steuer** [comment]
thanks ingo, fixed

**Comment by p-steuer** [comment]
merged.

**Comment by dodys** [comment]
Thanks @p-steuer!

**Pull request merged**